modules/social_features/social_event/modules/social_event_invite/social_event_invite.install

```diff
@@ -197,3 +197,45 @@ function social_event_invite_update_11401(): string {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Update email invitation messages.
+ */
+function social_group_invite_update_11601(): void {
+  $config_factory = \Drupal::configFactory();
+
+  $group_config = \Drupal::config('social_group.settings');
+  $invite_settings = $group_config->get('group_invite');
+
+  $configurations = [
+    'group.content_type.closed_group-group_invitation',
+    'group.content_type.flexible_group-group_invitation',
+    'group.content_type.open_group-group_invitation',
+    'group.content_type.public_group-group_invitation',
+    'group.content_type.secret_group-group_invitation',
+  ];
+
+  foreach ($configurations as $name) {
+    $configuration = $config_factory->getEditable($name);
+
+    if ($configuration->isNew()) {
+      // Configuration isn't exist.
+      continue;
+    }
+
+    // Set email subject text.
+    if (
+      !empty($invite_settings['invite_subject'])
```

Comment:
Shouldn't we check if the configuration for the group has already been set before overriding it with our own? Open Source users might have altered this themselves already and we are throwing away their changes.

Reply:
> 
- Added "Solution" section
- Changed milestone to 11.5.5